test/utils.test.js

```diff
@@ -19,3 +19,13 @@ describe('isInput', () => {
     expect(isInput({ tagName: 'A' })).to.be.false;
   });
 });
+
+describe('keyNameVals', () => {
+  it('returns the key value of the given key name', () => {
+    expect(keyNameVals('left')).to.be.equal('ArrowLeft');
+    expect(keyNameVals('up')).to.be.equal('ArrowUp');
+    expect(keyNameVals('right')).to.be.equal('ArrowRight');
+    expect(keyNameVals('down')).to.be.equal('ArrowDown');
+    expect(keyNameVals('numlock')).to.be.equal('numlock');
```

Comment:
Shouldn't this be `Numlock` as of the [spec](https://www.w3.org/TR/DOM-Level-3-Events-key/)?